src/SignaturePad.Forms.Platform.Shared/ColorExtensions.cs

```diff
@@ -15,13 +15,17 @@
 using Android.Widget;
 using Xamarin.Forms.Platform.Android;
 using NativeColor = Android.Graphics.Color;
+#elif NET471
```

Comment:
I think it may be better to declare a `WINDOWS_DESKTOP` define and use that - if we ever change the .net version, then we will have to update code all over the place.

Reply:
NETFRAMEWORK variable goes by default. I use it